style.css

```diff
@@ -129,4 +137,171 @@ header {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.recent-projects-parent {
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: center;
+    gap: 30px;
+    max-width: 800px;
+}
+
+.project-card {
+    flex-basis: 33%;
+    flex-grow: 1;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+}
+
+.project-card img {
+    object-fit: cover;
+    height: 30vh;
+}
+
+.project-card a {
+    text-decoration: none;
+    font-size: 1.5rem;
+    color: #ff8000;
+    transition: all 0.5s;
+}
+
+.project-card a:hover {
+    color: #001453;
+    text-decoration: underline;
+}
+
+.more-projects-link, .contact-form-submit-button {
+    margin: 50px 20px;
+    padding: 25px 45px;
+    color: #001453;
+    border: 2px solid #001453;
+    border-radius: 5px;
+    background-color: #ffffff;
+    border-radius: 3px;
+    font-size: 1.5rem;
+    cursor: pointer;
+}
+
+.more-projects-link:hover {
+    color: #ffffff;
+    background-color: #ff8000;
+    border-color: #ff8000;
+}
+
+.contact-form-submit-button:hover {
+    color: #ffffff;
+    background-color: #ff8000;
+    border-color: #ffffff;
+}
+
+.contact-section {
+    padding: 50px 20px;
+    background-color: #ff8000;
+    color: #ffffff;
+    display: flex;
+    flex-flow: column wrap;
+    align-items: center;
+    justify-content: center;
+}
+
+.contact-header {
+    display: flex;
+    flex-flow: column wrap;
+    align-items: center;
+    justify-content: center;
+}
+
+.contact-section h2 {
+    margin-top: 0;
+}
+
+.contact-form {
+    display: flex;
+    flex-flow: column wrap;
+    align-items: center;
+    justify-content: center;
+    gap: 25px;
+    width: 400px;
+    font-size: 1.5rem;
+}
+.contact-form div{
+    width: 100%;
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+}
+
+.contact-input {
+    display: inline-block;
+    border: none;
+    border-radius: 2px;
+}
+
+input {
+    width: 190px;
+    min-height: 35px;
+
+}
+textarea {
+    width: 100%;
+}
+
+footer {
+    background-color: #ffffff;
+    padding: 40px 0;
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: center;
+}
+
+footer div {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: center;
+    gap: 25px;
+}
+
+.footer-item a {
+    color:#001453;
+    font-size: 1.5rem;
+    text-transform: uppercase;
+    font-weight: 700;
+    letter-spacing: 0.25rem;
+}
+
+.footer-item a:hover {
+    color: #bdbdbd;
+}
+
+@media screen and (max-width: 600px) {
+    h1 {
+        font-size: 7rem;
+    }
+
+    h2 {
+        font-size: 5rem;
+    }
+    .nav-bar{
+        flex-direction: column;
+    }
+    .recent-projects-parent {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+	}
+    .project-card {
+        padding: 20px 0;
+        border-bottom: 1px solid #DDDDDD;
+        align-items: center;
+        width: 80%;
```

Comment:
best to have full width and to create space on the sides using either padding or margins.

if using padding, you might also have to add:

```
box-sizing: border-box;
```

we normally add that on all elements, rather than just one. then you would have to check that it hasn't broken any of your other elements